### ISSUE TITLE
InputWrapper is missing classes for when the Input inside of it is invalid (aria-invalid)

### ISSUE THREAD

**Issue by viczam** [keenthemes/reui#16]

**Comment by oziriemeka** [comment]
Happy to provide a fix for this, and provide continous contributions to this project

**Comment by sean0205** [comment]
Resolved: [Changelog](https://www.reui.io/docs/changelog#v1021---21-august-2025)
